File: 01. SourceCode/ChildFund/InformationHub.Model/SearchResultObject.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace InformationHub.Model.SearchResults
{
    public class SearchResultObject<T>
    {
        public int TotalItem { get; set; }

        public List<T> ListResult = new List<T>();
        public List<string> ListId = new List<string>();

        public string PathFile { get; set; }
        //exten
        public int TotalItemStatus6 { get; set; }
        public int TotalItemStatus1 { get; set; }
    }
}

File: 01. SourceCode/Android/ChildProfile/app/src/main/java/nts/childprofile/model/SearchResultObject.java
package nts.childprofile.model;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by NTS-VANVV on 27/12/2018.
 */

public class SearchResultObject<T> {
    public int TotalItem;

    public List<T> ListResult = new ArrayList<>();

    public String PathFile;
}
